src/services/item/plugins/app/appSetting/service.ts

```diff
@@ -53,11 +78,22 @@ export class AppSettingService {
     // patching requires admin rights
     await this.itemService.get(member, repositories, itemId, PermissionLevel.Admin);
 
-    const appSetting = await appSettingRepository.get(appSettingId);
-
-    await this.hooks.runPreHooks('patch', member, repositories, appSetting);
-
-    return appSettingRepository.patch(itemId, appSettingId, body);
+    // const appSetting = await appSettingRepository.get(appSettingId);
```

Comment:
Remove.

Reply:
Now that I think about it, maybe update operations should have the hooks `args` set to both `{ original: item, update: Item }`. But I've just checked and somehow the items are missing the `update` hooks, oops 🤔